### PR TITLE
feat: page break in SoA pdf

### PR DESCRIPTION
**Pull request by rtdany10**
- [X] Fetch customer even if no email is found.
- [X] Ability to add page break inside PDF after each SoA when printed consolidated.

https://user-images.githubusercontent.com/52111700/196178937-807e7db1-f277-4636-8c4e-425823f71e54.mp4

#no-docs

### CODE REVIEW

File: erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py [hunk 1]
```diff
@@ -240,8 +242,8 @@ def fetch_customers(customer_collection, collection_name, primary_mandatory):
 		if int(primary_mandatory):
 			if primary_email == "":
 				continue
-		elif (billing_email == "") and (primary_email == ""):
-			continue
+		# elif (billing_email == "") and (primary_email == ""):
```
Comment (deepeshgarg007) [inline erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py]: Why is this commented out?

Comment (rtdany10) [reply erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py]: This line of code would continue the iteration if no email was found. Email is necessary only when I send it via email, not when I download the consolidated PDF. Hence, removed this code and added validation on email sending function

Comment (deepeshgarg007) [reply erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py]: We usually don't leave any commented out code, remove if its not needed

Comment (rtdany10) [reply erpnext/accounts/doctype/process_statement_of_accounts/process_statement_of_accounts.py]: Updated :)

**Comment by rtdany10** [comment]
@mergifyo backport-version-13

**Comment by rtdany10** [comment]
@Mergifyio backport version-13-hotfix

**Comment by rtdany10** [comment]
@deepeshgarg007 could you back port this as well, please?

**Pull request merged**